cmake/CMakeLists.txt

```diff
@@ -1483,6 +1474,50 @@ install(
   DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/profile.d
 )
 
+###############################################################################
+# Install LAMMPS lib and python module into site-packages folder with
+# "install-python" target.  Behaves exactly like "make install-python" for
+# conventional build.  Only available, if a shared library is built.
+# This is primarily for people that only want to use the Python wrapper.
+###############################################################################
+if(BUILD_LIB AND BUILD_SHARED_LIBS)
+  find_package(PythonInterp)
+  if (${PYTHONINTERP_FOUND})
+    add_custom_target(
+      install-python
+      ${PYTHON_EXECUTABLE} install.py -v ${LAMMPS_SOURCE_DIR}/version.h
+      -m ${CMAKE_CURRENT_SOURCE_DIR}/../python/lammps.py
+      -l ${CMAKE_BINARY_DIR}/liblammps${CMAKE_SHARED_LIBRARY_SUFFIX}
+      WORKING_DIRECTORY  ${CMAKE_CURRENT_SOURCE_DIR}/../python
+      COMMENT "Installing LAMMPS Python module")
+  else()
+    add_custom_target(
+      install-python
+      echo "Must have Python installed to install the LAMMPS Python module")
+  endif()
+else()
+  add_custom_target(
+    install-python
+    echo "Must build LAMMPS as a shared library to use the Python module")
+endif()
+
+###############################################################################
+# Add LAMMPS python module to "install" target. This is taylored for building
+# LAMMPS for package managers and with different prefix settings.
+# This requires either a shared library or that the PYTHON package is included.
+###############################################################################
+if((BUILD_LIB AND BUILD_SHARED_LIBS) OR (PKG_PYTHON))
+  find_package(PythonInterp)
+  if (${PYTHONINTERP_FOUND})
+    if(NOT PYTHON_INSTDIR)
```

Comment:
`PYTHON_INSTDIR` is not a real cached variable maybe it is time to make it one and have it default to the value below.